deeppavlov/skills/aiml_skill/aiml_skill.py

```diff
@@ -118,6 +118,10 @@ def __call__(self, utterances_batch: List, history_batch: List, states_batch: Li
         # In this implementation we use current datetime for generating uniqe ids
         output_states_batch = []
         user_ids = []
+        if not states_batch:
```

Comment:
`if states_batch is None` would look cleaner since we set `None` as a default value.
I believe a possibility of empty lists should be handled separately for `utterances_batch` only.